orienteer-core/src/main/java/org/orienteer/core/boot/loader/util/JarUtils.java

```diff
@@ -164,37 +233,4 @@ public boolean accept(File dir, String name) {
         }
         return fullClassName;
     }
-
-    public Set<String> getAllClassNamesInModule(Path pathToJar) throws IOException {
```

Comment:
Agree with this. Goal of this one is unclear and logic seems to be questionable.